samples/advanced/content-security-policy.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=Edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta http-equiv="Content-Security-Policy" content="default-src 'self'">
+	<title>Scriptable > Bubble | Chart.js sample</title>
+	<link rel="stylesheet" type="text/css" href="../../dist/Chart.min.css">
+	<link rel="stylesheet" type="text/css" href="./content-security-policy.css">
+	<script src="../../dist/Chart.min.js"></script>
+	<script src="../utils.js"></script>
+	<script src="content-security-policy.js"></script>
+</head>
+<body>
+	<div class="content">
+		<div class="wrapper">
```

Comment:
Maybe put a short explanation here like "This page loads CSS while setting a strict content security policy"

Reply:
![image](https://user-images.githubusercontent.com/3874900/52400216-cd6aeb80-2abe-11e9-81b4-8a83486dbb5a.png)

Reply:
I like it!